inc_internal/zt_internal.h

```diff
@@ -47,6 +47,8 @@ typedef void (*send_cb)(int status, void *ctx);
 
 typedef void (*ch_connect_cb)(ziti_channel_t *ch, void *ctx, int status);
 
+typedef void (*ch_notify_state)(ziti_channel_t *ch, ziti_router_status status, void *ctx);
```

Comment:
maybe keep the parameter order the same? ish? keep the order the same as the ch_connect_cb ? (flip ctx/status)

Reply:
it is internal API, consistency is not required. Also I prefer `ctx` parameter to go last